docs/designs/helmify.md

```diff
@@ -0,0 +1,44 @@
+# Helm-ify
+
+## The problem
+
+Currently the project relies on helm install-ing several helm Charts,
+required for the various components. 
+However, other parts of the project such as the project components themselves 
+and migrations are a combination of `kustomize.yaml` manifests (components), k8s
+ yamls applied through `draconctl` (migrations). This is significantly confusing
+and tech debt since it is not clear what installs what, how and why.
+
+It is suggested that the project uses Helm as its sole way of deploying things.
+
+## Changes suggested
+
+The following changes are suggested
+
+* Make a `dracon-dev` chart which installs all the sub-charts referenced by 
+the `make-dev` target
+* Make `draconctl migrations apply` only apply migrations against a database url
+and not try to manage a k8s job
+* Make a `dracon` chart which installs dracon components and migrations only
```

Comment:
The migrations should be applied by the chart that deploys the database where the migrations will be applied

Reply:
there is no individual database chart. Either we install postgres as part of the dracon-dev values.dev.yaml or we just run the migrations as a dracon post-install job against the database string identified in the values.yaml